user/articles/130-web-experience-management/03-content-management/04-content-sets/01-intro.markdown

```diff
@@ -0,0 +1,17 @@
+# Introduction to Content Sets
+
+A Content Set is exactly what it sounds like: a set of content items. In @product-ver@, an administrator can define a list of content, and then that list of content can be displayed somewhere. The way that the Content Set is displayed is determined by the method that is used to display it. For example, if the Content Set is being used by a smartwatch app, it could be displayed as a simple list of titles, and selecting a title would cause the full article to display on a connected mobile device. The same Content Set could be displayed in a web browser with the full content of each article.
+
+In previous versions of the Liferay, the Asset Publisher was your most powerful tool for delivering content to users. With the Asset Publisher, administrators define either static lists of assets to display to a user or dynamic lists based on criteria like tags, categories, or asset type. In @product-ver@ Content Sets take the core idea of defining different types of asset lists and expands it. Content Lists are created outside of the context of a specific application or widget and can be used and re-used across different channels and applications.
+
+## Creating and Displaying Content Sets
```

Comment:
I'm confused now, this article is almost the same as the previous one. 
So I went back to it and that other one is written within the info framework directory. :/

Reply:
It looks like there was a merge issue and multiple versions of files with different names somehow got pushed up to the branch.